robottelo/ui/hosts.py

```diff
@@ -77,6 +93,21 @@ def _configure_interface_parameters(self, parameters_list):
                 'host',
                 (parameter_name.lower()).replace(' ', '_')
             ))]
+            # send_keys() can't send left parenthesis (see
+            # SeleniumHQ/selenium#674), which is a part of network type name
+            # (e.g. 'Physical (Bridge)')
+            if parameter_name == 'Network type' and ' (' in parameter_value:
```

Comment:
seems as pretty nasty solution to have it implemented each time we face the issue. Could you please think about some kind of a helper?